use-timescale/troubleshoot-timescaledb.md

```diff
@@ -100,6 +100,46 @@ equal the sum of `timescaledb.max_background_workers` and
 
 For more information, see the [worker configuration docs][worker-config].
 
+### Cannot compress chunk
+
+You might see this error message when trying to compress a chunk if
+the ACL for the compressed hypertable is corrupt:
+
+```sql
+tsdb=> SELECT compress_chunk('_timescaledb_internal._hyper_65_587239_chunk');
+ERROR: role 149910 was concurrently dropped
+```
+
+This can be caused if you dropped a user for the hypertable before
+TimescaleDB 2.5. For this case, the user would be removed from
```

Comment:
Not 100% sure:

```suggestion
TimescaleDB 2.5. In this case, the user would be removed from
```

Reply:
FYI, I think this is the bug that fixes this issue: https://github.com/timescale/timescaledb/pull/3625